docs/assemblies/proc_creating-a-set-of-data-plane-nodes.adoc

```diff
@@ -0,0 +1,272 @@
+  nodeTemplate:
+    ...
+    ansible:
+      ansibleVars:
+        edpm_network_config_template: |
+              ---
+              network_config:
+              - type: interface
+                name: nic1
+                mtu: {{ ctlplane_mtu }}
+                dns_servers: {{ ctlplane_dns_nameservers }}
+                domain: {{ dns_search_domains }}
+                routes: {{ ctlplane_host_routes }}
+                use_dhcp: false
+                addresses:
+                - ip_netmask: {{ ctlplane_ip }}/{{ ctlplane_subnet_cidr }}
+                {% for network in networks_all if network not in networks_skip_config %}
+                {% if network not in ["External", "Tenant"] and network in role_networks %}
+                - type: interface
+                  name: nic{{ loop.index +1 }}
+                  mtu: {{ lookup('vars', networks_lower[network] ~ '_mtu') }}
+                  use_dhcp: false
+                  addresses:
+                  - ip_netmask:
+                    {{ lookup('vars', networks_lower[network] ~ '_ip') }}/{{ lookup('vars', networks_lower[network] ~ '_cidr') }}
+                  routes: {{ lookup('vars', networks_lower[network] ~ '_host_routes') }}
+                {% elif network in role_networks or 'external_bridge' in role_tags %}
+                - type: ovs_bridge
+                {% if network == 'External' %}
+                  name: {{ neutron_physical_bridge_name }}
+                {% else %}
+                  name: {{ 'br-' ~ networks_lower[network] }}
+                {% endif %}
+                  mtu: {{ lookup('vars', networks_lower[network] ~ '_mtu') }}
+                  dns_servers: {{ ctlplane_dns_nameservers }}
+                  use_dhcp: false
+                  addresses:
+                  - ip_netmask:
+                    {{ lookup('vars', networks_lower[network] ~ '_ip') }}/{{ lookup('vars', networks_lower[network] ~ '_cidr') }}
+                  routes: {{ lookup('vars', networks_lower[network] ~ '_host_routes') }}
+                  members:
+                  - type: interface
+                    name: nic{{loop.index + 1}}
+                    mtu: {{ lookup('vars', networks_lower[network] ~ '_mtu') }}
+                    use_dhcp: false
+                    primary: true
+                {% endif %}
+                {% endfor %}
+----
++
+You can copy a sample template from https://github.com/openstack-k8s-operators/edpm-ansible/tree/main/roles/edpm_network_config/templates. For information about customizing the template, see link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/17.1/html/installing_and_managing_red_hat_openstack_platform_with_director/assembly_configuring-overcloud-networking_installing-director-on-the-undercloud#ref_network-interface-configuration-options_custom-network-interface-templates[Network interface configuration options].
+
+. If your nodes are not pre-provisioned, you must configure the bare-metal template:
```

Comment:
Same here...

Reply:
it is just a quick recap, right below it points the link with the full explanation

Reply:
"If your nodes are not pre-provisioned, you must configure the bare-metal template" could be reworded as it' confusing. It could say  "Set the `baremetalSetTemplate` section in `OpenStackDataPlaneNodeSet` spec as required and just point to the detailed doc "Provisioning bare-metal data plane nodes". We probably should not repeat the sample here. A few more things..

1. I would prefer we write it as `bare metal` and not `bare-metal` in the text.
2. Also it would be good to have all NodeSet specific sections including bare metal provisioning before talking about deployment (i.e proc_deploying-the-data-plane.adoc)

Reply:
I don't think repeating the sample is necessary either, in fact it could make things more painful for us down the line when we'll have to update docs for some reason. Regarding terminology, the primary criterion would be what are we calling things elsewhere. 

If we are using wrong terms everywhere, we need to file an issue. If we do not, we shouldn't introduce more cases.

Reply:
I've addressed the comments: https://logserver.rdoproject.org/48/648/a36b23a90e9c40e53bde42a9706bc6ed9450f0ad/github-check/dataplane-operator-docs-preview/a9d9959/docs_build/docs_build/dataplane/index-upstream.html